packages/plugins/Solana/src/state/IdentityService.ts

```diff
@@ -1,18 +1,21 @@
 import type { Plugin } from '@masknet/plugin-infra'
-import { IdentityServiceState } from '@masknet/plugin-infra/web3'
 import { SocialIdentity, SocialAddress, NetworkPluginID, SocialAddressType } from '@masknet/web3-shared-base'
+import { IdentityServiceState } from '@masknet/plugin-infra/web3'
+
+const SOL_RE = /(?=\w)\S{1,256}\.sol\b/
+const ADDRESS_FULL = /(?!0x)\w{44}/
```

Comment:
This will match `11FPVZjSLJn2NSjWwu5yhMyHMwE3Ty974E9aCyZngxNM8a` (I add `11` before an address string).